python/cudf_polars/cudf_polars/callback.py

```diff
@@ -145,7 +146,14 @@ def _callback(
         set_device(device),
         set_memory_resource(memory_resource),
     ):
-        return ir.evaluate(cache={}).to_polars()
+        if executor is None or executor == "cudf":
```

Comment:
```suggestion
        if executor is None or executor == "pylibcudf":
```

Reply:
Done in https://github.com/rapidsai/cudf/pull/17262/commits/21e598af17a8755c09f8e14aa18b858830ba329e .